postgres/cloudpostgres/cloudpostgres.go

```diff
@@ -55,6 +58,45 @@ type Params struct {
 	TraceOpts []ocsql.TraceOption
 }
 
+// URLOpener opens GCP PostgreSQL URLs
+// like "cloudpostgres://user:password@myproject/us-central1/instanceId/mydb".
+type URLOpener struct {
+	// TraceOpts contains options for OpenCensus.
+	TraceOpts  []ocsql.TraceOption
+	CertSource *certs.RemoteCertSource
```

Comment:
Using a non-interface type here implies a specific implementation. Instead, use `proxy.CertSource`.

```suggestion
	CertSource proxy.CertSource
```

Reply:
(And this field should have a doc comment.)